main.py

```diff
@@ -0,0 +1,41 @@
+"this module is a template"
```

Comment:
this should be removed i think since this is now a module and not a template anymore

Reply:
Fixed in https://github.com/R2D2-2019/vision/commit/d473d24737d5ffb7655a90050e2977153ae85cad